src/Tizen.Uix.Stt/Tizen.Uix.Stt/ErrorOccurredEventArgs.cs

```diff
@@ -182,12 +181,11 @@ public string ErrorMessage
                 if (error != SttError.None)
                 {
                     Log.Error(LogTag, "GetErrorMessage Failed with error " + error);
-                    return "";
+                    throw ExceptionFactory.CreateException(error);
```

Comment:
X AVOID throwing exceptions from property getters. 

    Property getters should be simple operations and should not have any preconditions. If a getter can throw an exception, it should probably be redesigned to be a method. Notice that this rule does not apply to indexers, where we do expect exceptions as a result of validating the arguments.